src/api/app/controllers/webui/packages/job_history_controller.rb

```diff
@@ -13,6 +13,8 @@ def index
       private
 
       def set_package
+        # Store the package name in case of multibuilds
```

Comment:
If this comment needed? :thinking: In the other commits I add the same fix without any comment. I would either add it in all the cases, or remove in all the cases. :thinking:

Reply:
I was planning to combine the setters for all package sub controllers. But for the cloud upload and rebuild times controller we want this method to behave differently. So this probably would make things to complicated.

For now I just aligned the code in `build_reason_controller.rb` to what I did in `job_history_controller.rb`